### ISSUE TITLE
How to install?

### ISSUE THREAD

**Issue by BLACK4585** [Edwardsoen/aw-watcher-steam#4]
I'm trying to install it, but regardless what I'm trying, the Steam Watcher never appears.
Could you please provide a complete Step-by-Step documentation?

**Comment by Edwardsoen** [comment]
Hi, the documentation is in Readme.md

**Comment by AidenMatthews1** [comment]
Having the same issue. Successfully installed through pip following the Readme.md but still not showing up when right clicking activity watch after restarting multiple times. I also tried the install procedure listed on the [utilisation watcher](https://github.com/Alwinator/aw-watcher-utilization?tab=readme-ov-file) after installing dependencies through pip but that also didnt work. 

The readme doesnt include actually running the script. After poking around in the code a little to my (limited) knowledge the relative imports used will not work with automatic start unless python is explicitly told about the project which it will on the devs machine. [related stack overflow explanation I used to understand code](https://stackoverflow.com/questions/16981921/relative-imports-in-python-3) 
The following error will be given when running the __init__ on any machine that does not have the install directory included in path: 
ImportError: attempted relative import with no known parent package

I managed to get a little further by manually going into powershell, navigating to #/ActivityWatch/aw-watcher-steam-master/aw_watcher_steam and entering "python __main__.py" which runs the script directly instead of through the init and changes some of the import settings allowing the script to get past the relative import problems. 

At this point I dont have steamguard setup so cant get my API key and test if this entirely fixes the problem but It seems to run the script correctly where before it would immediately quit with an import error. The only downside is needing to manually run the script on startup but that should be able to be automated through windows.

**Comment by Milliw** [comment]
Copy this.
The installation procedure is lacking quite some information...

I managed to git clone this repo and then run the pip install command inside the directory.
The script then tells it has successfully installed the watcher but I don't see any folder on my device that contains similar files like the other watchers. Watcher does not get loaded on aw start.

I'm on Linux if that's important.

**Comment by Edwardsoen** [comment]
i will take a look at this issue and update the readme file this weekend

**Comment by marcinsmialek** [comment]
I can't get it to work too, despite that it looks like installed.  
No rules match its timeline logs.

I'm using Python 3.12 on Windows with PDM, and I cloned the repo.

Installing as global from the repo's root folder didn't work: `pdm install -g`  
However it worked from the parent folder: `pdm install -g -p .\aw-watcher-steam\`

After ActivityWatch restart, there was a plugin config to set up - uncomment and fill the values

```powershell
ii "$env:LOCALAPPDATA\activitywatch\activitywatch\aw-watcher-steam\aw-watcher-steam.toml"
```  

My `aw-watcher-steam.toml` (redacted)

```toml
[aw-watcher-steam]
steam_id = "12345678901234567"
api_key = "000102030405060708090A0B0C0D0E0F"
poll_time = 5.0
```

Autostarting plugin - add `aw-watcher-steam` to the `aw-qt.toml` autostart list, and uncomment it:

```powershell
ii "$env:LOCALAPPDATA\activitywatch\activitywatch\aw-qt\aw-qt.toml"
```

My `aw-qt.toml`

```toml
[aw-qt]
autostart_modules = ["aw-server", "aw-watcher-afk", "aw-watcher-window", "aw-watcher-steam"]
```

aw-watcher-steam appears in the timeline view and it shows the game I was playing - name (HELLDIVERS™ 2) and id. Same with Darktide.

But I can't find a way to add a rule that would match the Steam games in general. No luck with rules matching a single game either - I tried both name and id.
ActivityWatch beta finds the game by itself by window name, but it's not the same.

**Comment by Edwardsoen** [comment]
> But I can't find a way to add a rule that would match the Steam games in general.

Hi,  sorry i dont quite understand what you are trying to do

**Comment by 95276324** [comment]
yeah, i had installed and cant do the step 3, i click  click on aw-watcher-steam several times but didnt work

**Comment by cate02** [comment]
Having trouble too. W11, furthest I got was main.py running and working but no module or bucket in activitywatch
I and thus probably many others would love clarification...
-When downloading the zip do we only keep the aw_watcher_steam and discard the other contents of the _master folder?
-A simple auto script to install all dependencies instead of messing with python and terminal would be much appreciated
-Where should the folder go? activitywatch/activitywatch/<here>? or appdata/local/<here>?
-Why is a toml config file created in activitywatch directory?
-...We were already supposed to input the steam information inside the main.py file, hardcoding it?
-Would be nice to have a little non-scary "your-steam-info-config.json" file
-After file's set, steam info is provided, dependencies sorted, main.py running and working, how do you get it to work with activitywatch? Doesn't show in modules or buckets

This would be super awesome if I was able to get it working, I hope you can return to this project one day :)

**Comment by 95276324** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
